frontend/drink_list_picker.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+
+
+async function login_user() {
+    let drink_holder = document.getElementById("drink_list_show");
+    
+
+    let response = await fetch(`menulist.json`);
+    let obiekt = await response.json();
+    for (let element of obiekt.coctails) {
+        let div = document.createElement("div");
+        let img = document.createElement("img");
+        let name = document.createElement("a")
+        name.textContent = element.name;
+        name.className = "drink_name"
+        div.className = "drink_element";
+        img.className = "drink_img";
+        img.src = element.link;
+        div.append(img);
+        div.append(name);
+        drink_holder.append(div);
+    }
+}
+login_user();
```

Comment:
File should end with an empty line, add an empty line after this line.

Reply:
This is a problem in other files too, so remember to add an empty line at the end of every file (keep in mind that there should be ONLY ONE empty line at the end of the file)

Reply:
ok

Reply:
You left some files still without an empty line at the end.
style.css
index.html
Please look through all of the files in the project :)